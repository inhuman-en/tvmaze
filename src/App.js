import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import './App.scss';

import { Header } from './navigation';

import AppRoutes from './AppRoutes';

const App = () => (
  <div className="App">
    <Router>
      <Header />
      <AppRoutes />
    </Router>
  </div>
);

export default App;
