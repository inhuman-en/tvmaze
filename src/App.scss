@import 'style/variables';

$padding: 15px;

main {
  max-width: 900px;
  margin: 0 auto;
  padding: 0 $padding;
}
