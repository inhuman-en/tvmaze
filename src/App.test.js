import { configure, shallow } from 'enzyme';
// TODO: consider moving to v17 adapter when it's out
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import App from './App';

configure({ adapter: new Adapter() });

describe('App component', () => {
  it('should render', () => {
    const result = toJson(shallow(<App />));
    expect(result).toMatchSnapshot();
  });
});
