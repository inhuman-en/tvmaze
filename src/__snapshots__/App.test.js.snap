// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render 1`] = `
<div
  className="App"
>
  <BrowserRouter>
    <Header />
    <main>
      <AppRoutes />
    </main>
  </BrowserRouter>
</div>
`;
